index.html

```diff
@@ -6,67 +6,141 @@
   <meta name="viewport" content="width=device-width, initial-scale=1.0">
   <script src="https://kit.fontawesome.com/f8112dc1f2.js" crossorigin="anonymous"></script>
   <link rel="stylesheet" href="style.css">
+  <link rel="preconnect" href="https://fonts.googleapis.com">
+  <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+  <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;900&display=swap" rel="stylesheet">
   <title>My Portfolio</title>
 </head>
 <body>
-  <header>
-    <a class="wrap logo" href="#">Kehinde Audu</a>
-    <nav>
-      <i class="fas fa-bars"></i>
-    </nav>
-  </header>
-  <main>
-    <section class="hero">
-      <h1>Kehinde Audu</h1>
-      <p class="tag">Software Developer</p>
-      <p class="decription">
-        I can help you build a product , feature or website Look through some of my work and experience! If you like 
-        what you see and have a project you need coded,  don’t hestiate to contact me.
-      </p>
-      <div class="abilities">
-        <div class="languages wrap">
-          <p>Languages</p>
-          <ul>
-            <li>JavaScript</li>
-            <li>Ruby</li>
-            <li>Html</li>
-            <li>CSS</li>
-          </ul>
+    <header>
+      <a class="wrap logo" href="#">Kehinde Audu</a>
+      <nav>
+        <i class="fas fa-bars"></i>
+      </nav>
+    </header>
+    <main>
+      <section class="hero">
```

Comment:
- [ optional ] It will be nice to give this section a background image as in the [Figma design](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=48%3A2346)